File: EduSafe.WebSite/src/app/classes/resultsEmailEntry.ts
import { ModelInputEntry } from './modelInputEntry';
import { ModelOutputSummary } from './modelOutputSummary';

export class ResultsEmailEntry {
  RecipientAddress: string;
  RecipientName: string
  ResultsPageHtml: string;
  ModelInputEntry: ModelInputEntry;
  ModelOutputSummary: ModelOutputSummary;
}

File: EduSafe.WebApi/Models/Individuals/ResultsEmailEntry.cs
namespace EduSafe.WebApi.Models.Individuals
{
    public class ResultsEmailEntry
    {
        public string RecipientAddress { get; set; }
        public string RecipientName { get; set; }
        public string ResultsPageHtml { get; set; }
        public ModelInputEntry ModelInputEntry { get; set; }
        public ModelOutputSummary ModelOutputSummary { get; set; }
    }
}